omnibus/files/private-chef-ctl-commands/cleanup_bifrost.rb

```diff
@@ -0,0 +1,213 @@
+require 'pg'
+
+CREATE_SQL =<<SQL
+BEGIN;
+CREATE TABLE IF NOT EXISTS cleanup_tracking_auth_actors(
```

Comment:
Should we have `TEMPORARY` or possible `TEMPORARY UNLOGGED`(if we don't want the table to replicate)?